lib/services/task.ts

```diff
@@ -47,4 +48,20 @@ export class TaskService {
       await db.tasks.update(id, updatedTask);
     }
   };
+
+  // update only the color of the task
+  // ease of use function for future on the frontend Task UI
+  // we still need to add base update functionality for the tasks
+  // but I see myself wanting to easily change the color of tasks without impacting anything else
```

Comment:
don't need this long comment - but make sure to comment the updateNoteColor cousin to this with a JS Doc comment too